### ISSUE TITLE
Conflict in rdf-spec dependency

### ISSUE THREAD

**Issue by jeremyf** [ActiveTriples/ActiveTriples#180]
In attempting to run the specs, I was encountering errors with the
    custom rdf-spec dependency. I suspect that the rdf-spec was updated
    since the last integration test that was run.
    
    ```console
    Bundler could not find compatible versions for gem "rdf":
      In Gemfile:
        active-triples (>= 0) ruby depends on
          rdf (~> 1.1.13) ruby
    
        rdf-spec (>= 0) ruby depends on
          rdf (~> 1.99) ruby
    ```

**Comment by jcoyne** [comment]
How is it you are getting this. See here: https://github.com/ActiveTriples/ActiveTriples/blob/develop/active-triples.gemspec#L17

**Comment by jcoyne** [comment]
Is this due to confusion between the develop and master branch?

**Comment by no-reply** [comment]
> @no-reply thinks gitflow is the bee's knees.

:laughing: I don't feel that strongly about it; but we've been using it here for a while; and it's in use on all the RDF.rb gems. I think we'd need a pretty strong argument to motivate a change.

In any case, this is closed by merge of #181.
